### DISCUSSION TITLE
mycroft mycroft_bus_client import MessageBusClient, Message

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#236] [Q&A] [answered]
does ovos have a replacment for 
from mycroft_bus_client import MessageBusClient, Message

the docs dont list it.  Thank you

**Comment by JarbasAl** [comment, marked as answer]
`ovos_bus_client` instead of mycroft_bus_client

https://github.com/OpenVoiceOS/ovos-bus-client

**Reply by geetee24** [reply]
same api interface?
